### PR TITLE
chore: Fix attaching desktop builds to tags

### PR DESCRIPTION
**Pull request by ffflorian**

### CODE REVIEW

File: electron/package.json [hunk 1]
```diff
@@ -7,9 +7,11 @@
   "devDependencies": {
     "@types/node": "12.19.16",
     "@wireapp/copy-config": "1.1.13",
+    "adm-zip": "0.5.2",
```
Comment (lipis) [inline electron/package.json:10]: Why not your own: https://github.com/ffflorian/jszip-cli :)

**Review by lipis — APPROVED** [review]

File: .github/workflows/test_build_deploy.yml [hunk 2]
```diff
@@ -421,24 +426,40 @@ jobs:
         working-directory: electron
         run: |
           echo "OS=macos" >> $GITHUB_ENV
-          yarn build:mac
+          yarn build:macos
+
+      - name: Zip Electron build
```
Comment (lipis) [inline .github/workflows/test_build_deploy.yml]:
```suggestion
      - name: Zip desktop build
```

**Comment by lipis** [comment]
I would change all `Desktop` to `desktop`..

File: electron/package.json [hunk 3]
```diff
@@ -34,7 +36,8 @@
     "dist": "yarn clear:ts && yarn build:ts",
     "postinstall": "copy-config",
     "prestart": "yarn dist",
-    "start": "electron . --no-sandbox"
+    "start": "electron . --no-sandbox",
+    "zip": "node bin/zip.js"
```
Comment (bennycode) [inline electron/package.json:40]: Let's make use of `ts-node`.

**Review by bennycode — CHANGES_REQUESTED** [review]

File: electron/package.json (hunk 3 above)
Comment (ffflorian) [reply electron/package.json:40]: I think it's too much overhead to use ts-node and a separate `tsconfig.json` for two files. Additionally, I am already using `// @ts-check`.

File: electron/package.json (hunk 1 above)
Comment (ffflorian) [reply electron/package.json:10]: Not the right use case. `jszip-cli` is supposed to be used as CLI.

**Pull request merged**